ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.2.5f1
+m_EditorVersion: 2018.2.18f1
```

Comment:
In general, this one shouldn't be committed except we explicitly want to bump our minimum version requirement.